ionic/src/app/add-devices/add-devices.component.ts

```diff
@@ -33,57 +33,63 @@ export class AddDevicesComponent implements OnInit {
     private api: ApiService,
     private deviceService: DeviceService,
     public alertController: AlertController,
-    public modalController: ModalController
+    public modalController: ModalController,
+    private navParm: NavParams
   ) { }
 
   ngOnInit() {
-    this.scanDevice();
+    this.getdataPairDevice()
   }
+  getdataPairDevice() {
+    const devicePingData = this.navParm.get('pingDevice')
+    this.devicePing = devicePingData['pingDevice']
+  }
+
   createLoginForm() {
     this.setNameForm = new FormGroup({
       name: new FormControl("", [
-        Validators.required,
-      ]),
+        Validators.required
+      ])
     });
   }
-  scanDevice() {
-    this.mode = "scan";
-    this.isScanningDevice = true;
-    this.wifinetworks = [];
-    this.devicePing = {
-      name: "",
-      chip: "",
-      webid: "",
-      isNew: false,
-      type: ""
-    }
-    this.keepCheckingWifiConnected();
-    this.router.navigate(["/add-devices"]);
-  }
-  keepCheckingWifiConnected() {
-    if (wifiCheckInterval)
-      clearInterval(wifiCheckInterval);
-    wifiCheckInterval = setInterval(async () => {
-      try {
-        const data = await this.api.checkPing();
-        this.devicePing = data['data']
-        if (this.devicePing.name.length > 0) {
-          this.devicePing.isNew = false;
-        } else {
-          this.devicePing.isNew = true;
-        }
-        this.isScanningDevice = false;
-        clearInterval(wifiCheckInterval);
-        this.mode = "discovery";
-      } catch (e) {
-        console.log(e)
-        this.isScanningDevice = true;
-        this.errorMessage = e["error"];
-        this.notifyService.alertUser("Device not found");
-      }
-    }, 5000);
-  }
-  async setDeviceName(name: String, chip: string, formData) {
+  // scanDevice() {
+  //   this.mode = "scan";
+  //   this.isScanningDevice = true;
+  //   this.wifinetworks = [];
+  //   this.devicePing = {
+  //     name: "",
+  //     chip: "",
+  //     webid: "",
+  //     isNew: false,
+  //     type: ""
+  //   }
+  //   this.keepCheckingWifiConnected();
+  //   this.router.navigate(["/add-devices"]);
+  // }
+  // keepCheckingWifiConnected() {
+  //   if (wifiCheckInterval)
```

Comment:
don't comment remove. in PR there should not be comments like this. unless there is a good reason